src/validators/gender_sum_validator.py

```
@@ -0,0 +1,386 @@
+        """Extract male, female, and total sections from source file.
+
+        Args:
+            source_path: Path to source CSV file
+
+        Returns:
+            Dict with 'male', 'female', 'total' keys containing data rows
+        """
+        import subprocess
+
+        # Shift_JIS から UTF-8 に変換
+        result = subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)